tests/test_idpi/conftest.py

```diff
@@ -32,7 +61,9 @@ def template_env():
 
 
 @pytest.fixture
-def fieldextra(tmp_path, data_dir, template_env, fieldextra_executable):
+def fieldextra(
+    tmp_path, data_dir, template_env, fieldextra_executable
+) -> Callable[..., xr.Dataset | List[xr.Dataset]]:
```

Comment:
```suggestion
) -> Callable[..., xr.Dataset | list[xr.Dataset]]:
```
the built in list is now generic